x/nodes/keeper/common_test.go

```diff
@@ -135,6 +133,17 @@ func sendFromModuleToAccount(t *testing.T, ctx sdk.Ctx, k *Keeper, module string
 	}
 }
 
+func fundAccount(ctx sdk.Context, k Keeper,
+	address sdk.Address, coin sdk.Coin) sdk.Error {
```

Comment:
NIT: Either keep everything on one line or multiline the whole thing
```suggestion
func fundAccount(
	ctx sdk.Context,
	k Keeper,
	address sdk.Address,
	coin sdk.Coin
) sdk.Error {
```